.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ["2.7", "3.6", "3.7"]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install dependencies
+        run: pip install -r requirements-test.txt
+      - name: Install RPM tools
+        run : |
+          curl --create-dirs -o ~/.local/bin/spectool https://pagure.io/rpmdevtools/raw/26a8abc746fba9c0b32eb899b96c92841a37855a/f/spectool.in
+          curl --create-dirs -o ~/.local/bin/rpmdev-bumpspec https://pagure.io/rpmdevtools/raw/6f387c1deaa5cbed770310e288abde04b17421dc/f/rpmdev-bumpspec
+          curl --create-dirs -o ~/.local/bin/rpmdev-packager https://pagure.io/rpmdevtools/raw/master/f/rpmdev-packager
+          chmod +x ~/.local/bin/spectool
+          chmod +x ~/.local/bin/rpmdev-bumpspec
+          chmod +x ~/.local/bin/rpmdev-packager
+          sudo apt-get install -y git-annex rpm
+      - name: Run tests
+        run: tox
+      - name: Run container tests
+        run: docker build --build-arg VERSION=master -t quay.io/foreman/obal:latest .
```

Comment:
This doesn't need to be part of the main testing. It builds the same container 3 times.

I'd even argue it's pointless to do this on PR since it only builds master.

Reply:
Yeah, it can be an own step.

Why it builds `master` I can't remember, maybe @ehelms can enlighten us?

Reply:
The PR SHA's are not fetched by default so pypi can't clone them. This is the next best thing. I think it would be better if you could pass in a wheel that was built and install that.

Reply:
well, at least that one is fixed now :)